analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -244,8 +342,13 @@ def construct_analyzer_cmd(self, result_handler):
                                      '-analyzer-disable-checker=' +
                                      ','.join(disabled_checkers)])
             # Enable aggressive-binary-operation-simplification option.
-            analyzer_cmd.extend(
-                clang_options.get_abos_options(config.version_info))
+            version_info = ClangSA.version_info()
+            if version_info and version_info.major_version >= 8:
+                analyzer_cmd.extend([
```

Comment:
I agree with making functions shorter. However, I'd implement this thing in #3817, because this "abos" thing is an analyzer config option which is used by default in CodeChecker. This should be handled like "HeaderFilterRegex" option of clang-tidy in the other PR.